ddp-workspace/projects/ddp-pancan/src/styles.scss

```diff
@@ -74,10 +74,111 @@ a {
     box-sizing: border-box;
 }
 
-.italic {
+.Italic {
     font-style: italic;
 }
 
 .Link {
     color: $primary-color;
 }
+
+.button-link {
+    border: none;
+    background: none;
+    font-size: inherit;
+    font-family: inherit;
+    color: $primary-color;
+
+    &:hover, &:focus {
+        text-decoration: underline;
+    }
+}
+
+.Modal-title {
+    @extend .static-page-title;
+
+    margin: 10px 0 12px 0;
+    display: flex;
+    justify-content: space-between;
+    position: relative;
+
+    .mat-icon-button {
+        top: -20px;
+        right: -15px;
+    }
+}
+
+.Modal-text {
+    font-size: 1.2rem;
+    line-height: 1.8rem;
+    margin-bottom: 15px;
+    margin-top: 0;
+}
+
+.Modal-progress {
+    margin: 5px 0;
+}
+
+.JoinDialogFiled {
+    width: 100%;
+}
+
+.JoinDialogNameField {
+    width: 50%;
+    margin: 0 30px 0 0;
+}
+
+.JoinDialogLastnameField {
+    width: 50%;
+}
+
+.JoinDialogGroupFields {
+    display: flex;
+}
+
+.JoinField--margin {
+    margin-bottom: 5px;
+}
+
+@media only screen and (max-width: 650px) {
+    .JoinDialogGroupFields {
+        flex-direction: column;
+    }
+
+    .JoinDialogNameField {
+        width: 100%;
+        margin: 0;
+    }
+
+    .JoinDialogLastnameField {
+        width: 100%;
+    }
+}
+
+.ButtonFilled {
+    @extend .btn-common;
+
+    outline: none;
+    padding: 0.7rem 1.5rem;
+    border: none;
+    color: #fff !important;
+    transition: all .3s ease-out;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,.16),0 2px 10px 0 rgba(0,0,0,.12) !important;
+    background-color: $primary-color;
+}
+
+.ButtonFilled:hover {
```

Comment:
Should we add `focus` event handler here as well?

Reply:
sure! done